### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by zhangt2333**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@zhangt2333 Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=20192)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-projects/spring-boot&pullRequestId=20192) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@zhangt2333 Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-projects/spring-boot&pullRequestId=20192)!

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jdbc/DataSourceInitializer.java [hunk 1]
```diff
@@ -125,10 +125,7 @@ private boolean isEnabled() {
 		if (mode == DataSourceInitializationMode.NEVER) {
 			return false;
 		}
-		if (mode == DataSourceInitializationMode.EMBEDDED && !isEmbedded()) {
-			return false;
-		}
-		return true;
+		return mode != DataSourceInitializationMode.EMBEDDED || isEmbedded();
```
Comment (snicoll) [inline spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/jdbc/DataSourceInitializer.java]: We prefer multi if statements to stay separate for better readability.

File: spring-boot-project/spring-boot-cli/src/main/java/org/springframework/boot/cli/command/core/HelpCommand.java [hunk 2]
```diff
@@ -81,10 +81,7 @@ public String getUsageHelp() {
 	}
 
 	private boolean isHelpShown(Command command) {
-		if (command instanceof HelpCommand || command instanceof HintCommand) {
-			return false;
-		}
-		return true;
+		return !(command instanceof HelpCommand) && !(command instanceof HintCommand);
```
Comment (snicoll) [inline spring-boot-project/spring-boot-cli/src/main/java/org/springframework/boot/cli/command/core/HelpCommand.java]: I find this harder to read

**Comment by zhangt2333** [comment]
@snicoll : It's OK now.

**Comment by snicoll** [comment]
@zhangt2333 thank you for making your first contribution to Spring Boot.

**Pull request closed without merging**